src/main/java/com/javacode/entities/Company.java

```diff
@@ -0,0 +1,85 @@
+package com.javacode.entities;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "companies")
+public class Company {
+	@Id
+	@Column(name = "id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@Column(name = "companyName")
+	private String companyName;
+
+	@Column(name = "address")
+	private String address;
+
+	@Column(name = "companyDescription")
+	private String companyDescription;
+
+	@OneToMany(mappedBy = "company") // we need to duplicate the physical information
+	private Set<Job> jobs;
```

Comment:
Dùng List thay vì Set em nhé,
Check và sửa lại cho toàn bộ quan hệ `oneToMany` nhé

Reply:
Em sửa rồi ạ